### PR TITLE
Caching, user feedback, accessibility

### PR DESCRIPTION
**Pull request by emishra**

### CODE REVIEW

File: source/reactjs/README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
+# MSApprovals
+MSApprovals is an important service and platform that supports 16000 users in approving ~750K approvals each month with the total amount ranging anywhere from ~3B$ to ~5B$ and this is across ~21 different Line of Business applications.
```
Comment (cr-chinmaya) [inline source/reactjs/README.md:2]: Let's use MS Assent and also swap this content for something which we already have in the other readme file.

File: source/reactjs/src/Components/Shared/SharedLayout.ts [hunk 2]
```diff
@@ -89,6 +89,15 @@ export const AccessibilityIcon = styled.div`
...
     font-size: 16;
+
+
+    @media screen and (forced-colors: active) and (prefers-color-scheme: light) {
+        background-image: url('/icons/ic_fluent_accessibility_24_regular.svg');
+    }
+
+    
+    
+
```
Comment (cr-chinmaya) [inline source/reactjs/src/Components/Shared/SharedLayout.ts:100]: Can we please remove the extra whitespace?

**Review by cr-chinmaya — CHANGES_REQUESTED** [review]

File: source/reactjs/src/Components/Shared/SharedLayout.ts (hunk 2 above)
Comment (emishra) [reply source/reactjs/src/Components/Shared/SharedLayout.ts]: removed whitespace

File: source/reactjs/README.md (hunk 1 above)
Comment (emishra) [reply source/reactjs/README.md]: I've updated the readme accordingly

Comment (cr-chinmaya) [reply source/reactjs/README.md]: Microsoft Assent (a.k.a Approvals) as a platform provides the "one stop shop" solution for approvers via a model that brings together disparate different approval requests in a consistent and ultra-modern model. Approvals delivers a unified approvals experience for any approval on multiple form factors - Website, Outlook Actionable email, Teams. It consolidates approvals across organization's line of business applications, building on modern technology and powered by Microsoft Azure. It serves as a showcase for solving modern IT scenarios using the latest technologies.

**Review by cr-chinmaya — APPROVED** [review]
Looks good!

**Review by v-ragjos — APPROVED** [review]

**Pull request merged**